experiment_scheduler/submitter/edit.py

```diff
@@ -37,6 +37,7 @@ def main():
     )
     stub = master_pb2_grpc.MasterStub(channel)
 
+    # pylint: disable=E1101
```

Comment:
It this pylint disabled necessary for all lines below or for a specific single line?
If second is right, could you add a comment at the single line?